src/BinSkim.Rules/RuleResources.resx

```diff
@@ -213,10 +213,10 @@ Modules triggering this check were:
     <value>'{0}' has the imports section marked executable. Because the loader will always mark the imports section as writable, it is important to mark this section as non-executable, so that an attacker cannot place shellcode here. To resolve this issue, ensure that your program does not mark the imports section as executable. Look for uses of /SECTION or /MERGE on the linker command line, or #pragma segment in source code, which change the imports section to be executable, or which merge the ".rdata" segment into an executable section.</value>
   </data>
   <data name="BA2010_Pass" xml:space="preserve">
-    <value>'{0}' does not have an imports section that is marked as executable, helping to prevent the exploitability of code vulnerabilities.</value>
+    <value>'{0}' does not have an imports section that is marked as executable, helping to prevent the exploitation of code vulnerabilities.</value>
```

Comment:
I think it should either be "prevent the exploitation of" (which is what I chose), or "reduce the exploitability of", but not "prevent the exploitability of".